src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9379,6 +9379,7 @@ public static partial class Directory
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("windows")]
         public static System.IO.DirectoryInfo CreateDirectory(string path, System.IO.UnixFileMode unixCreateMode) { throw null; }
         public static System.IO.FileSystemInfo CreateSymbolicLink(string path, string pathToTarget) { throw null; }
+        public static System.IO.DirectoryInfo CreateTempSubdirectory(string? prefix = null) { throw null; }
```

Comment:
Was it actively decided to add `Sub` in this name? I prefer the slightly shorter: `CreateTempDirectory`.

Reply:
Me too but in the review, it was mentioned that it is actually a sub-directory within the temp folder. So...

Reply:
> Was it actively decided to add `Sub` in this name? I prefer the slightly shorter: `CreateTempDirectory`.

Yes, that was the decision in the [API review](https://youtu.be/nrpWy7kmZ28?t=2365). The reasoning is that it is more explicit. "Create Temp Directory" makes it sound like it is creating the actual Temp directory, and not a directory inside the Temp directory.